karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
This file is the MAGIC KARMA FILE!  
This pulls all the JS into a headless browser, and then runs any tests it finds in those files for those files. :smile: